pkg/kubelet/kubelet.go

```diff
@@ -1834,6 +1834,24 @@ func (kl *Kubelet) GetPods() []*api.Pod {
 	return kl.podManager.GetPods()
 }
 
+// GetRunningPods returns all pods running on kubelet from looking at the
+// container runtime cache. This function converts kubecontainer.Pod to
+// api.Pod, so only the fields that exist in kubecontainer.Pod are considered
```

Comment:
Modified it to make it clear that only common fields (of kubecontainer.Pod and api.Pod) are meaningful.